patches/protobufjs+6.11.3.patch

```diff
@@ -1,50 +0,0 @@
-diff --git a/node_modules/protobufjs/src/util/minimal.js b/node_modules/protobufjs/src/util/minimal.js
-index 3c406de..083f71c 100644
---- a/node_modules/protobufjs/src/util/minimal.js
-+++ b/node_modules/protobufjs/src/util/minimal.js
-@@ -280,13 +280,38 @@ function newError(name) {
-             merge(this, properties);
-     }
- 
--    (CustomError.prototype = Object.create(Error.prototype)).constructor = CustomError;
--
--    Object.defineProperty(CustomError.prototype, "name", { get: function() { return name; } });
--
--    CustomError.prototype.toString = function toString() {
--        return this.name + ": " + this.message;
--    };
-+    CustomError.prototype = Object.create(Error.prototype, {
```

Comment:
Was this protobuf patch deleted because we're now depending on a protobuf in which this bug has been fixed?

Reply:
@michaelfig ^

Reply:
With this PR, we're now depending on our agoric-labs fork of protobufjs, which has the bugfix.